osu.Game/Screens/Play/HUD/ArgonHealthDisplay.cs

```diff
@@ -92,36 +78,42 @@ private void load()
         {
             AutoSizeAxes = Axes.Y;
 
-            InternalChild = new Container
+            InternalChild = content = new Container
             {
-                AutoSizeAxes = Axes.Both,
                 Children = new Drawable[]
                 {
-                    background = new BackgroundPath
+                    new ArgonHealthDisplayBackground
                     {
+                        RelativeSizeAxes = Axes.Both,
                         PathRadius = MAIN_PATH_RADIUS,
+                        PathPadding = MAIN_PATH_RADIUS
                     },
-                    glowBar = new BarPath
+                    new Container
                     {
-                        BarColour = Color4.White,
-                        GlowColour = main_bar_glow_colour,
-                        Blending = BlendingParameters.Additive,
-                        Colour = ColourInfo.GradientHorizontal(Color4.White.Opacity(0.8f), Color4.White),
-                        PathRadius = 40f,
-                        // Kinda hacky, but results in correct positioning with increased path radius.
-                        Margin = new MarginPadding(-30f),
-                        GlowPortion = 0.9f,
+                        RelativeSizeAxes = Axes.Both,
+                        Padding = new MarginPadding(-30f),
```

Comment:
Bringing back removed comment:
```suggestion
                        // Kinda hacky, but results in correct positioning with increased path radius.
                        Padding = new MarginPadding(-30f),
```

Reply:
I don't think it's hacky, so I'll change the comment a bit, but sure

Reply:
Reads better.